testing/terraform/main.tf

```diff
@@ -64,11 +64,38 @@ locals {
   ec2_instance_profile = var.ec2_instance_profile
 }
 
+resource "aws_security_group" "ec2_sec_group" {
+  name_prefix = "daemon-test-sg-"
+  description = "Allow HTTP and SSH traffic via Terraform"
+
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
Not sure if it is possible, but can we limit the IP addresses more?

Reply:
That would be ideal. However, we are running terraform from a GitHub runner whose IP could be anything, so I had to keep this ingress open.